src/setup/error.py

```diff
@@ -16,9 +16,9 @@
 
 from cortx.utils.errors import BaseError
 
-class SetupError(BaseError):
+class RgwSetupError(BaseError):
```

Comment:
This can be called "SetupError" unless there is a specific part of the code which is RGW specific. 
Try to write as generic code as possible.

Reply:
Renamed to 'SetupError'